src/configure/configure.ts

```diff
@@ -255,14 +255,12 @@ class Orchestrator {
     }
 
     private async analyzeNode(node: any): Promise<GenericResource> {
-        if (!!node) {
-            return await this.extractAzureResourceFromNode(node);
-        }
-        else if (node && node.fsPath) {
-            this.workspacePath = node.fsPath;
-            telemetryHelper.setTelemetry(TelemetryKeys.SourceRepoLocation, SourceOptions.CurrentWorkspace);
+        if (!!node && !await this.extractAzureResourceFromNode(node)) {
```

Comment:
lets extract the filepath thing also in a similar function to make the code simpler to read.

```
if (!!node) {
    resourceNodeFound = await this.extractAzureResourceFromNode(node);
    if(!resourceNodeFound) {
       await this.extractFolderInfoFromNode(node);
    }
}
```